Sources/Build/BuildPlan.swift

```diff
@@ -2303,8 +2321,19 @@ private func generateResourceInfoPlist(
     return true
 }
 
-fileprivate extension TSCUtility.Triple {
-    var isSupportingStaticStdlib: Bool {
+extension TSCUtility.Triple {
+    fileprivate var isSupportingStaticStdlib: Bool {
         isLinux() || arch == .wasm32
     }
 }
+
+extension ResolvedPackage {
+    fileprivate var isRemote: Bool {
+        switch self.underlyingPackage.manifest.packageKind {
+        case .registry, .remoteSourceControl, .localSourceControl:
+            return true
+        case .root, .fileSystem:
+            return false
```

Comment:
Just to confirm, this would make `path` dependencies be considered local and not remote, right?  If so, I completely agree, that's seems like the right semantics.

Reply:
exactly